lib/rollout.rb

```diff
@@ -1,28 +1,30 @@
 require "rollout/version"
 require "zlib"
 require "set"
+require "json"
 
 class Rollout
   class Feature
-    attr_accessor :groups, :users, :percentage
+    attr_accessor :groups, :users, :percentage, :data
     attr_reader :name, :options
 
     def initialize(name, string = nil, opts = {})
       @options = opts
       @name    = name
 
       if string
-        raw_percentage,raw_users,raw_groups = string.split("|")
+        raw_percentage,raw_users,raw_groups,raw_data = string.split("|")
         @percentage = raw_percentage.to_f
         @users = (raw_users || "").split(",").map(&:to_s).to_set
         @groups = (raw_groups || "").split(",").map(&:to_sym).to_set
+        @data = JSON.parse(raw_data || "{}", symbolize_names: true)
       else
         clear
       end
     end
 
     def serialize
-      "#{@percentage}|#{@users.to_a.join(",")}|#{@groups.to_a.join(",")}"
+      "#{@percentage}|#{@users.to_a.join(",")}|#{@groups.to_a.join(",")}|#{@data.to_json}"
```

Comment:
in case of @data equals to nil

```
[119] pry(main)> nil.to_json
=> "null"
```

then when you load feature next time you will get

```
[120] pry(main)> JSON.parse("null")
JSON::ParserError: 784: unexpected token at 'null'
```

---------

Can you please address this issue and store just empty string in case `@data` is blank. Also can you please add spec to test this issue?